src/core/pusher.ts

```diff
@@ -68,6 +68,11 @@ export default class Pusher {
         `You should always specify a cluster when connecting. ${suffix}`
       );
     }
+    if (options.disableStats) {
```

Comment:
You might also want a check like

```js
if (options.enableStats !== undefined && options.disableStats !== undefined) {
   Logger.error('Ambiguous: remove deprecated disableStats from your config')
}
```
```

Reply:
Not sure how strongly you feel about this specific error message.

I replicated the existing behavious in 6e53ba4 but this specific error message is missing. I don't feel it's necessary but if you think it is, I can add it.